health-services/individual/src/main/java/org/egov/individual/service/IndividualService.java

```diff
@@ -38,55 +60,168 @@ public class IndividualService {
 
     private final IndividualRepository individualRepository;
 
+    private final List<Validator<IndividualBulkRequest, Individual>> validators;
+
+    private final ObjectMapper objectMapper;
+
+    private final IndividualProperties properties;
+
+    private final Predicate<Validator<IndividualBulkRequest, Individual>> isApplicableForUpdate = validator ->
+            validator.getClass().equals(NullIdValidator.class)
+                    || validator.getClass().equals(IsDeletedValidator.class)
```

Comment:
Good suggestion but since we have made it generic, it won't be possible because the logic which is iterating over validators is agnostic of create, update or delete as we are taking the applicability parameter as a predicate which we can call on any validator.